src/main/scala/dsptools/numbers/chisel_concrete/DspReal.scala

```diff
@@ -107,6 +107,14 @@ class DspReal(lit: Option[BigInt] = None) extends Bundle {
     oneOperandOperator(Module(new BBFCeil()))
   }
 
+  def round(): DspReal = (this + DspReal(0.5)).floor()
```

Comment:
DspReal has some things that call the typeclass, especially truncate. That seemed bad. This may change the current behavior, though, because the implementation of truncate here doesn't have pipeline registers.